src/datasource.js

```diff
@@ -117,22 +117,24 @@ export default class OCIDatasource {
         queryType: 'search',
         region: _.isEmpty(region) ? this.defaultRegion : region,
         compartment: compartmentId,
-        namespace: namespace
+        namespace: namespace,
+        resourcegroup: resourcegroup
       }],
       range: this.timeSrv.timeRange()
     }).then((res) => {
       return this.mapToTextValue(res, 'search')
     })
   }
 
-  /** 
+  /**
    * Build and validate query parameters.
    */
-  async buildQueryParameters(options) {
+  async buildQueryParameters (options) {
     let queries = options.targets
       .filter(t => !t.hide)
       .filter(t => !_.isEmpty(this.getVariableValue(t.compartment, options.scopedVars)) && t.compartment !== SELECT_PLACEHOLDERS.COMPARTMENT)
       .filter(t => !_.isEmpty(this.getVariableValue(t.namespace, options.scopedVars)) && t.namespace !== SELECT_PLACEHOLDERS.NAMESPACE)
+      .filter(t => !_.isEmpty(this.getVariableValue(t.resourcegroup, options.scopedVars)) && t.resourcegroup !== SELECT_PLACEHOLDERS.RESOURCEGROUP)
```

Comment:
@japatel
instead of
` .filter(t => !_.isEmpty(this.getVariableValue(t.resourcegroup, options.scopedVars)) && t.resourcegroup !== SELECT_PLACEHOLDERS.RESOURCEGROUP)`
Please use 
` .filter(t => !_.isEmpty(this.getVariableValue(t.resourcegroup, options.scopedVars)).`
either we can not create a quiery

Reply:
@VadimKomar In the the new commit I have reverted the ResourceGroup placeholder to 'select resourcegroup'. With this new change we are able to see graphs for both namespaces with or without ResoruceGroup.

Reply:
We will still need .filter(t => !_.isEmpty(this.getVariableValue(t.resourcegroup, options.scopedVars)) && t.resourcegroup !== SELECT_PLACEHOLDERS.RESOURCEGROUP)